cmd/jaeger/internal/extension/jaegerquery/server.go

```diff
@@ -79,18 +79,23 @@ func (s *server) Start(_ context.Context, host component.Host) error {
 	if err != nil {
```

Comment:
the issue mentioned above is partially solved, at least for metrics and traces, which is what I need. You should find a way how to configure internal telemetry

https://github.com/open-telemetry/opentelemetry-collector/blob/main/docs/observability.md#how-we-expose-telemetry

Reply:
Ah okay, will look into that.